example-context/example-service/example-event/example-event-processor/pom.xml

```diff
@@ -35,74 +39,4 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
-
-
-    <build>
-        <finalName>${project.artifactId}</finalName>
-        <plugins>
-            <plugin>
-                <groupId>uk.gov.justice.maven</groupId>
-                <artifactId>raml-maven-plugin</artifactId>
-                <version>${raml-maven-plugin.version}</version>
-                <executions>
-                    <execution>
-                        <id>generate-adapter</id>
-                        <goals>
-                            <goal>generate</goal>
-                        </goals>
-                        <phase>generate-sources</phase>
-                        <configuration>
-                            <generatorName>uk.gov.justice.raml.jms.core.JmsEndpointGenerator</generatorName>
-                            <sourceDirectory>${basedir}/src/raml</sourceDirectory>
-                            <outputDirectory>${project.build.directory}/generated-sources</outputDirectory>
-                            <basePackageName>uk.gov.justice.api</basePackageName>
-                            <includes>
-                                <include>**/*cakeshop-event-processor.messaging.raml</include>
-                            </includes>
-                        </configuration>
-                    </execution>
-                    <execution>
-                        <id>generate-client</id>
-                        <goals>
-                            <goal>generate</goal>
-                        </goals>
-                        <phase>generate-sources</phase>
-                        <configuration>
-                            <generatorName>uk.gov.justice.services.clients.messaging.generator.MessagingClientGenerator</generatorName>
-                            <sourceDirectory>${basedir}/src/raml</sourceDirectory>
-                            <outputDirectory>${project.build.directory}/generated-sources</outputDirectory>
-                            <basePackageName>uk.gov.justice.api</basePackageName>
-                            <includes>
-                                <include>**/*public-event.messaging.raml</include>
```

Comment:
I think it is the same as other configurations, the public-event raml will be included in the normal *.messaging.raml pick up